lib/src/dynamic_color_builder.dart

```diff
@@ -49,6 +49,11 @@ class DynamicColorBuilderState extends State<DynamicColorBuilder> {
 
   // Platform messages are asynchronous, so we initialize in an async method.
   Future<void> initPlatformState() async {
+    // If the widget was removed from the tree while the asynchronous platform
+    // message was in flight, we want to discard the reply rather than calling
+    // setState to update our non-existent appearance.
+    if (!mounted) return;
+
     CorePalette? corePalette;
```

Comment:
I like the style you used better than what's here. Can you please rewrite `corePalette` fetching and `setState`-ing  in a single try-on block, like you did in lines 71-85?

Reply:
I think I addressed this hopefully.